bucket/euler.json

```diff
@@ -0,0 +1,53 @@
+{
+    "homepage": "http://euler-math-toolbox.de",
+    "version": "2019-06-27",
+    "license": "GPL-3.0-or-later",
+    "description": "A powerful math program combining numerical and symbolic tools (using the computer algebra system Maxima).",
+    "architecture": {
+        "64bit": {
+            "url": "https://downloads.sourceforge.net/project/eumat/Installer/EulerSetup-64_2019-06-27.exe",
+            "hash": "sha1:dd83403978788848c9178b309d3c4d85bb56ad7d",
+            "installer": {
+                "script": [
+                    "Expand-InnoArchive -Path \"$dir\\$fname\"",
+                    "Expand-InnoArchive -Path \"$dir\\$fname\" -ExtractDir '{sys}' -Removal",
+                    "if (!(Test-Path \"$persist_dir\\.euler.profile\")) {",
+                    "   New-Item -Path \"$dir\\.euler.profile\" -ItemType 'File' | Out-Null",
+                    "}"
+                ]
+            }
+        }
+    },
+    "bin": [
+        "euler64.exe",
```

Comment:
You have architecture property and still 64bit shortcuts and binaries are out of it.

Reply:
Okay, move into arch.

The behavior should be the same, since 32bit platform will be blocked by 64bit only url.